README.md

```diff
@@ -1,6 +1,6 @@
 # Benchmarking
 
-The benchmark can be run [here](http://somebee.github.io/todos-vdom-bench/index.html)
+The benchmark can be run [here](http://somebee.github.io/dom-reconciler-bench/index.html)
```

Comment:
You don’t need the `index.html` btw

Reply:
Haha of course, didn’t even think about that.....